### PR TITLE
Add README badge for code coverage

### LINKED ISSUE #248: Add README badge for code coverage
NIJobs has a pretty good test coverage of all its features but we're not showcasing it to everyone. It'd be nice to have a badge showing how much coverage we have.

One could look into [Codecov](https://about.codecov.io/) or a similar tool. We should be able to easily integrate it with our Github actions. CI. You can look at multiple projects using this, but here's an example: https://github.com/codecov/example-javascript.

It's also possible to integrate this tool with pull requests and it'll tell us how the coverage has increased or decreased. We can consider using this if we think is worth it.

### PR DESCRIPTION
**Pull request by ttoino**
Closes #248.

### CODE REVIEW

**Comment by ttoino** [comment]
Code coverage reports can now be found [here](https://app.codecov.io/gh/NIAEFEUP/nijobs-be) (they will only show up for this pull request until it is merged).

Anyone from the organization should be able to login to Codecov with Github and access all the repository settings. I've left them all as the default, but we should discuss if we want any of them changed. [Here](https://docs.codecov.com/docs/common-recipe-list) are some common configurations and [here](https://docs.codecov.com/docs/codecovyml-reference) is the config file reference.

**Review by Naapperas — APPROVED** [review]
LGTM, there is a PR that fixes the audit issues, this will be merged after that

File: docker-compose.yml
Comment (BrunoRosendo) [inline docker-compose.yml:34]: I'm curious, any reason to use the `app/` folder instead of `nijobs-be` like the other volumes?

**Review by BrunoRosendo — COMMENTED** [review]
Good job! This is already approved, I just had a small question

File: docker-compose.yml
Comment (ttoino) [reply docker-compose.yml:34]: The [dockerfile](https://github.com/NIAEFEUP/nijobs-be/blob/c272e3bbbda9937b4ec7c2c4d8ac6bbb811684c7/Dockerfile-test#L4) is using the `app` folder as the workdir, not sure why the other volumes use `nijobs-be` :thinking:

Comment (BrunoRosendo) [reply docker-compose.yml:34]: You're right :upside_down_face: Seems like those volumes aren't even used... Can you fix this? The `web-dev` service is correct, this is the incorrect one

**Review by BrunoRosendo — DISMISSED** [review]
Nice! We just need to wait for #246 to be merged now

**Review by Naapperas — APPROVED** [review]
LGTM

**Pull request merged**
